examples/custom_sensor/Odometer.cc

```diff
@@ -44,16 +44,18 @@ bool Odometer::Load(const sdf::Sensor &_sdf)
   // Advertise topic where data will be published
   this->pub = this->node.Advertise<gz::msgs::Double>(this->Topic());
 
-  if (!_sdf.Element()->HasElement("ignition:odometer"))
+  if (_sdf->HasAttribute("gz:odometer"))
```

Comment:
This is a change in the implementation, is it necessary?

Reply:
Oops, fixed https://github.com/gazebosim/gz-sensors/pull/235/commits/79537ced942baf780b0556889b0f50a7c74568ff